### PR TITLE
(internal) release_minor: build v5.8.0

### PR DESCRIPTION
**Pull request by Chickensoupwithrice**
Test plan: automated release PR, CI will perform additional checks

### CODE REVIEW

**Comment by sourcegraph-buildkite** [comment]
- :green_circle: Internal release is ready for promotion!
- :warning: Do not close/merge the pull request or delete the associated branch if you intend to promote it.

File: prepare-release.sh [hunk 1]
```diff
@@ -45,7 +45,7 @@ fi
 
 echo "Retrieving latest tag..."
 git fetch
-latest_tag=$(git describe --tags --abbrev=0 | tr -d '\n' | cut -c2-)
+latest_tag="5.6.0"
```
Comment (eseliger) [inline prepare-release.sh:48]: might want to revert

Comment (Chickensoupwithrice) [reply prepare-release.sh]: Good catch

**Review by DaedalusG — APPROVED** [review]

**Comment by sourcegraph-buildkite** [comment]
- :green_circle: Internal release is ready for promotion!
- :warning: Do not close/merge the pull request or delete the associated branch if you intend to promote it.

**Pull request merged**